server/games/game.py

```diff
@@ -664,8 +665,9 @@ def clear_slot(self, slot_index):
     def getGamemodVersion(self):
         return self.game_service.game_mode_versions[self.game_mode]
 
+    # Allowing latin names only, and truncating them to be less than 64 characters long
     def sanitize(self, name):
-        return re.sub('[^\x20-\xFF]+', '_', name)
+        return re.sub('[^\x20-\xFF]+', '_', name)[0:64]
```

Comment:
The database length is 128. Why choose 64?

Reply:
128 is hella long for a game name, in best cases I think 64 would be enough. But I can make it [0:127] if you prefer.

Reply:
I thought you do this to prevent games not being saved. So yes, the limit should be the database limit.